bitbag/wishlist-plugin/4.4/post-install.txt

```diff
@@ -0,0 +1,2 @@
+  If you want to contact us, the best way is to fill the form on our website or send us an e-mail to <comment>hello@bitbag.io</comment> with your question(s).
```

Comment:
This post-install note should be removed.

Reply:
This is a standard we use in our recipes: https://github.com/symfony/recipes-contrib/tree/main/bitbag

Why do you think it shouldn't be included here?